### PR TITLE
finished project

### PR DESCRIPTION
**Pull request by yochiwarez**

### CODE REVIEW

**Comment by ebukaume** [comment]
Hi @yochiwarez :v: 

You have really done well so far; however, there are some important things you need to fix before you get a green.

Firstly, stickler-CI wasn't run for this PR. Did you set up stickler properly? You can use [these](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) to troubleshoot to make sure things are set up correctly.

**[REQUIRED]**
1. The elements in the navbar are supposed to be buttons. Why did you make them paragraphs? 
2. The 4 images at the bottom of the page are supposed to also be links, even if they link to '#'.
3. For the footer, see bellow
 - [ ] It should be wrapped by <footer> tags
 - [ ] It includes links like copyright, terms of use, and privacy policy

Please submit another code review request form after you have made the required changes.

Kindly see bellow more of my comments...

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        Apple
+    </title>
+</head>
+
+<body>
+    <main>
+        <div class="content">
+            <div>
+                <div class="nav-bar">
+                    <p><a href="#"><i class="fa fa-apple" aria-hidden="true"></i></a></p>
```
Comment (ebukaume) [inline index.html]: Are `<p>` tags the most semantic for this case?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    top: 200px;
+    width: 1000px;
+    margin: 0 auto;
+}
+
+.content div div.headline > p{
+    text-align: center;
+    color: white;
+    margin-bottom: 40px;
+    font-weight: 1;
```
Comment (ebukaume) [inline style.css]: 1 is not a correct value for font-weight. Please correct this.

**Comment by yochiwarez** [comment]
I disabled the hover style for the buttons

**Pull request merged**